test/src/org/omegat/core/dictionaries/LingvoDSLTest.java

```diff
@@ -86,28 +80,29 @@ public void testReadArticleRussian() throws Exception {
 
     @Test
     public void testReadArticleChinese() throws Exception {
-        LingvoDSLDict dict = (LingvoDSLDict) new LingvoDSL().loadDict(TEST_DICT, ENGLISH);
+        LingvoDSLDict dict = (LingvoDSLDict) new LingvoDSL().loadDict(TEST_DICT);
         String word = "\u4e00\u4e2a\u6837";
         List<DictionaryEntry> result = dict.readArticles(word);
         assertFalse(result.isEmpty());
         assertEquals(word, result.get(0).getWord());
-        assertEquals("&#91;y\u012B ge y\u00E0ng&#93;&nbsp;\nsame as \u4E00\u6A23|\u4E00\u6837 y\u012B y\u00E0ng&nbsp;, the same\n", result.get(0).getArticle());
+        assertEquals("[y\u012B ge y\u00E0ng]&nbsp;\nsame as \u4E00\u6A23|\u4E00\u6837 y\u012B " +
+                "y\u00E0ng&nbsp;, the same\n", result.get(0).getArticle());
     }
 
     @Test
     public void testReadArticleFontStyles() throws Exception {
-        LingvoDSLDict dict = (LingvoDSLDict) new LingvoDSL().loadDict(TEST_DICT, ENGLISH);
+        LingvoDSLDict dict = (LingvoDSLDict) new LingvoDSL().loadDict(TEST_DICT);
         String word = "italic";
         List<DictionaryEntry> result = dict.readArticles(word);
         assertFalse(result.isEmpty());
         assertEquals(word, result.get(0).getWord());
-        assertEquals("Here is an <span style='font-style: italic'>italic</span> <strong>word</strong>.\n",
+        assertEquals("<p>Here is an <span style='font-style: italic'>italic</span> <strong>word</strong>.</p>\n",
```

Comment:
Here is difference. When parser visit `[/m]`, we put `</p>` for `[m1]` - `[m9]` so we put `<p>` also for `[m]`